modules/site-health/admin-site-health.php

```diff
@@ -108,6 +108,96 @@ function ( &$value, $key ) {
 		return implode( ' | ', $array );
 	}
 
+	/**
+	 * Transform the option value to readable human sentence.
+	 *
+	 * @since 3.3
+	 *
+	 * @param string                $key option name.
+	 * @param string|array|bool|int $value option value.
+	 *
+	 * @return string|array $value option value.
+	 */
+	public function format_value( $key, $value ) {
+		$db_value = $value;
+		switch ( $key ) {
+			case 'browser':
+				if ( true === $value ) {
+					// translators: the placeholder is the option stored in database.
+					$value = __( 'Browser prefered language activated (%s)', 'polylang' );
```

Comment:
Looks like using `%s` with boolean false values will give you this : 
**Browser prefered language deactivated ()**